includes/class-wc-siftscience-html.php

```diff
@@ -27,6 +27,119 @@ class WC_SiftScience_Html {
 		public const WC_CHECKBOX_ELEMENT   = 'checkbox';
 		public const WC_SECTIONEND_ELEMENT = 'sectionend';
 		public const WC_CUSTOM_ELEMENT     = 'custom';
+		
+		public function __call( $name, $args ) {
+			if ( 'create_element' === $name ) {
```

Comment:
This will never happen because you already have a function named `create_element`

Reply:
call is triggered accissing thing from out side  [admin calls this function]
meaning it works i tested it